app/models/manageiq/providers/hawkular/middleware_manager/refresh_parser.rb

```diff
@@ -71,7 +71,7 @@ def parse_datasource(server, datasource, config)
           :nativeid          => datasource.id,
           :ems_ref           => datasource.path
         }
-        unless config.empty? || config['value'].empty?
+        unless config.empty? || config['value'].empty? || !config['value'].is_a?(Hash)
```

Comment:
How about converting that to a positive `if` statement, which tends to be easier to read?

And what happens if value contains a, say, `HashWithIndiffrentAccess` ?
You could check if `value` responds to `select`